pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[project]
+name = "commcare-sync"
+version = "0.1.0"
+description = "CommCare Sync simplifies the management of your CommCare data pipeline"
+readme = "README.md"
+requires-python = "~=3.10.0"  # TODO: ">=3.10,<3.15"
+dependencies = [
+    "Django",
+    "django-allauth",
+    "psycopg2-binary",
+    "djangorestframework",
+    "coreapi",  # DRF schemas and docs
+    "django-anymail[mailgun]",  # email services
+    "celery-progress",
+    "celery",
+    "celery[redis]",
```

Comment:
Are these distinct requirements? Is `"celery[redis]"` not enough on its own?

My understanding of `dep[req]` is that it means "Install `dep` along with it's optional set of dependencies indicated by `req`."

Reply:
75d8f33